main.cc

```diff
@@ -271,7 +217,7 @@ read_config(bpo::variables_map& opts, db::config& cfg) {
             }
             startlog.log(level, "{} : {}", msg, opt);
         });
-        co_await read_object_storage_config(cfg);
+        read_object_storage_config(cfg);
```

Comment:
Once db::config::object_storage_config is removed (in favor of db::config::object_storage_endpoints), this call will all be removed